src/net/sourceforge/kolmafia/request/UseItemRequest.java

```diff
@@ -5877,6 +5877,42 @@ else if (responseText.contains("You can't figure out the angle")) {
           return;
         }
         break;
+
+      case ItemPool.ROBY_RATATOUILLE_DE_JARLSBERG:
+      case ItemPool.ROBY_JARLSBERGS_VEGETABLE_SOUP:
+      case ItemPool.ROBY_ROASTED_VEGETABLE_OF_J:
+      case ItemPool.ROBY_PETES_SNEAKY_SMOOTHIE:
+      case ItemPool.ROBY_PETES_WILY_WHEY_BAR:
+      case ItemPool.ROBY_PETES_RICH_RICOTTA:
+      case ItemPool.ROBY_BORIS_BEER:
+      case ItemPool.ROBY_HONEY_BUN_OF_BORIS:
+      case ItemPool.ROBY_BORIS_BREAD:
+      case ItemPool.ROBY_CALZONE_OF_LEGEND:
+      case ItemPool.ROBY_PIZZA_OF_LEGEND:
+      case ItemPool.ROBY_ROASTED_VEGETABLE_FOCACCIA:
+      case ItemPool.ROBY_PLAIN_CALZONE:
+      case ItemPool.ROBY_BAKED_VEGGIE_RICOTTA:
+      case ItemPool.ROBY_DEEP_DISH_OF_LEGEND:
+      case ItemPool.PLANS_FOR_GRIMACITE_HAMMER:
+      case ItemPool.PLANS_FOR_GRIMACITE_GRAVY_BOAT:
+      case ItemPool.PLANS_FOR_GRIMACITE_WEIGHTLIFTING_BELT:
+      case ItemPool.PLANS_FOR_GRIMACITE_GRAPPLING_HOOK:
+      case ItemPool.PLANS_FOR_GRIMACITE_NINJA_MASK:
+      case ItemPool.PLANS_FOR_GRIMACITE_SHINGUARDS:
+      case ItemPool.PLANS_FOR_GRIMACITE_ASTROLABE:
+      case ItemPool.FETTUCINI_EPINES_INCONNU_RECIPE:
+      case ItemPool.SLAP_AND_SLAP_AGAIN_RECIPE:
+      case ItemPool.FUMBLE_FORMULA:
+      case ItemPool.MOTHERS_SECRET_RECIPE:
```

Comment:
@Veracity0 These changes are great but question: do we need to add to this switch now for every new recipe? Can we not derive this from parsing modifiers.txt?